bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
What would flatpak do differently if unsharing the user namespace doesn't work?

As far as I can see, it can either fail hard, or fall back to a reduced (less container-like, more chroot-like) level of containment. If you want it to work on RHEL, it sounds as though failing hard is not desired... is there anything else it can do, other than the "graceful degradation" that it already does?